Assets/Scripts/EnemyController.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class EnemyController : MonoBehaviour
+{
+    public double enemyHealth = 0f;
+    public Transform player;
+    public float attackRecharge;
+    public virtual void Attack ()
```

Comment:
możesz według mnie tu zakończyć tą metodę znakiem ';' zamiast robić niepotrzebnie więcej linijek za pomocą '{ }'

Reply:
takie tam czepianie się, ale racja można 2 linijek kodu oszczędzić hah

Reply:
poza tym jak teraz patrzę, to nawet mi C# nie pozwala tego zmienić, więc zostawiam :)